lib/livebook_web/router.ex

```diff
@@ -81,6 +81,7 @@ defmodule LivebookWeb.Router do
       live "/hub/:id/secrets/edit/:secret_name", Hub.EditLive, :edit_secret, as: :hub
 
       live "/sessions/:id", SessionLive, :page
+      live "/sessions/:id/external-window", SessionLive.ExternalWindowLive
```

Comment:
I was thinking to generalize the "popout" behaviour.  So maybe in the future this endpoint can be used for other functionality. E.g. pop out a code cell